Dockerfile

```diff
@@ -0,0 +1,38 @@
+# See examples/README.md to see how to build and run this
+
+FROM ubuntu:16.04
+
+# Uncomment the next 2 lines if you're behind a proxy:
+# ENV http_proxy="http://proxy.example.com:8080"
+# ENV https_proxy=${http_proxy}
```

Comment:
I thought about that, but decided not to because I don't see how it'll ever be used. Is there a case you're thinking of where it would be?